app/services/marketing_site.rb

```diff
@@ -60,10 +60,6 @@ def self.help_url
     URI.join(BASE_URL, locale_segment, 'help/').to_s
   end
 
-  def self.security_url
-    URI.join(BASE_URL, locale_segment, 'security/').to_s
-  end
```

Comment:
This was no longer used after #8508 and the URL is no longer correct either (login.gov/security now redirects to login.gov/policy)